Parsers/ASimProcessEvent/ARM/ASimProcessCreateLinuxSysmon/README.md

```diff
@@ -15,4 +15,4 @@ For more information, see:
 
 <br>
 
-[![Deploy to Azure](https://aka.ms/deploytoazurebutton)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2FAzure-Sentinel%2Fmaster%2FParsers%2FASimProcessEvent%2FARM%2FASimProcessCreateLinuxSysmon%2FASimProcessCreateLinuxSysmon.json) [![Deploy to Azure Gov](https://aka.ms/deploytoazuregovbutton)](https://portal.azure.us/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2FAzure-Sentinel%2Fmaster%2FParsers%2FASimProcessEvent%2FARM%2FASimProcessCreateLinuxSysmon%2FASimProcessCreateLinuxSysmon.json)
```

Comment:
I did not find what changed in this line (as well as the same line in other README.md files). While they are not critical I am always weary of accidental changes. Can you help me find what's differnent here?

Reply:
It is just whitespace differences.
I am using "Hide whitespace" checkbox:
![image](https://user-images.githubusercontent.com/46525261/166413063-9994bce9-9570-4398-a9a3-e24dc66d16e0.png)